src/TuDa.CIMS.Web/Components/WorkingGroupPage/WorkingGroupPurchaseList.razor.cs

```diff
@@ -55,12 +56,16 @@ private static string FormatCompletionDate(PurchaseResponseDto? value)
     private async Task NavigateToPurchase(PurchaseResponseDto purchase)
     {
         var options = new DialogOptions { CloseOnEscapeKey = true };
-
+        var signatureOrError = await _iPurchaseApi.RetrieveSignatureAsync(WorkingGroupId, purchase.Id);
+        if (signatureOrError.IsError)
+        {
+        }
+        string signature = signatureOrError.Value;
```

Comment:
🔧 🤔 We can just set an default value
```suggestion
        var signatureOrError = await _iPurchaseApi.RetrieveSignatureAsync(WorkingGroupId, purchase.Id).Else(<Some Value>)
```
Probably some value can be `""`